flake.nix

```diff
@@ -220,15 +218,15 @@
           # for which we don't apply the full build matrix such as cross or static.
           inherit (nixpkgsFor.${system}.native)
             changelog-d;
-          default = self.packages.${system}.nix-ng;
+          # TODO probably should be `nix-cli`
```

Comment:
or perhaps a tested `nix-cli`, like `nix-everything` but without the libs or manual?

We should probably also add a `nix` attribute with some suggestions of what to pick instead.